xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 12.10.2
+JSONRPC_VERSION 12.11.2
```

Comment:
If you bump minor, my understanding is patch should go back to 0

Reply:
That's my understanding too. Reset micro version (patch) to 0.

Reply:
your right my oversight done